doc/Command-Reference.md

```diff
@@ -4088,6 +4092,170 @@ This command deletes the SNMP Trap server IP address to which SNMP agent is expe
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#management-vrf)
 
+## Muxcable
+
+### Muxcable Show commands
+
+**show muxcable status**
+
+This command displays all the status of either all the ports which are connected to muxcable or any individual port selected by the user. The resultant table or json output will show the current status of muxcable on the port(auto/active) and also the health of the muxcable.
+
+- Usage:
+  ```
+  show muxcable status [OPTIONS] [PORT]
+  ```
+While displaying the muxcable status, users can configure the following fields
+1) optional - Port name should be a valid port
+2) optional - --json option to display the result in json format. By default output will be in tabular format.
+
+With no optional argument, all the ports muxcable status will be displayed in tabular form, or user can pass --json option to display in json format
+
+- Example:
+  ```
+  admin@sonic:~$ show muxcable status
+  PORT        STATUS    HEALTH
+  ----------  --------  --------
+  Ethernet32  active    HEALTHY
+  Ethernet0   auto      HEALTHY
+
+  admin@sonic:~$ show muxcable status --json
+  {
+      "MUX_CABLE": {
+	  "Ethernet32": {
+	      "STATUS": "active",
+	      "HEALTH": "HEALTHY"
+	  },
+	  "Ethernet0": {
+	      "STATUS": "auto",
+	      "HEALTH": "HEALTHY"
+	  }
+      }
+  }
+
+  admin@sonic:~$ show muxcable status Ethernet0
+  PORT       STATUS    HEALTH
+  ---------  --------  --------
+  Ethernet0  auto      HEALTHY
+
+  admin@sonic:~$ show muxcable status Ethernet0 --json
+  {
+      "MUX_CABLE": {
+	  "Ethernet0": {
+	      "STATUS": "auto",
+	      "HEALTH": "HEALTHY"
+	  }
+      }
+  }
+  ```
+
+**show muxcable config**
+
+This command displays all the configurations of either all the ports which are connected to muxcable or any individual port selected by the user. The resultant table or json output will show the current configurations of muxcable on the port(active/standby) and also the ipv4 and ipv6 address of the port as well as peer TOR ip address with the hostname.
+
+- Usage:
+  ```
+  show muxcable config [OPTIONS] [PORT]
+  ```
+With no optional argument, all the ports muxcable configuration will be displayed in tabular form
+While displaying the muxcable configuration, users can configure the following fields
+1) optional - Port name should be a valid port
+3) optional - --json option to display the result in json format. By default output will be in tabular format.
```

Comment:
Rather than `1)` and `3)`, I think you should use the option names

Reply:
fixed